src/transformer/events/mod_quiz/question_answered/essay.php

```diff
@@ -51,7 +51,7 @@ function essay(array $config, \stdClass $event, \stdClass $questionattempt, \std
         ],
         'timestamp' => utils\get_event_timestamp($event),
         'result' => [
-            'response' => $responsesummary,
+            'response' => ($responsesummary ? $responsesummary:''),
```

Comment:
This could be simplified a bit by using `?:`.

```suggestion
            'response' => ($responsesummary ?: ''),
```

Reply:
OK - made this change